internal/graphql/resolver/data-plane.go

```diff
@@ -41,18 +43,23 @@ func (r *Resolver) listenToDataPlaneState(ctx context.Context, provider models.P
 
 		for {
 			select {
-			case <-r.MeshSyncChannel:
+			case <-dataPlaneSyncChannel:
+				r.Log.Info("Dataplane sync channel called")
```

Comment:
Need to do a type check here, or else it will be triggered for other types too.

Reply:
We were using `MeshSyncChannel` for trigger, right. We should use it here too.

Also, you might need to update other places too that uses the `MeshSyncChannel` channel to use broadcast instead. (including handler ([here](https://github.com/meshery/meshery/blob/master/handlers/handler_instance.go#L29)) and Meshery-extension ([here](https://github.com/meshery/meshery/blob/de283e40e3b1477c79e64fdac7a4d1b7de8ad082/handlers/extensions.go#L44)) too).

Reply:
Yeah, do we want to move this meshsync channel to the broadcaster? If so, there'll be a lot of changes for this @dhruv0000